### PR TITLE
LibWeb: Convert CSS Token to new String classes

### PR DESCRIPTION
**Pull request by AtkinsSJ**
String, string, string, string, everybody loves string!

### CODE REVIEW

File: Userland/Libraries/LibWeb/CSS/Parser/Tokenizer.cpp [hunk 1]
```diff
@@ -356,18 +356,17 @@ Token Tokenizer::create_value_token(Token::Type type, DeprecatedString value)
...
-    token.m_value = move(value);
+    token.m_value = FlyString::from_utf8(value.view()).release_value_but_fixme_should_propagate_errors();
     return token;
 }
 
 Token Tokenizer::create_value_token(Token::Type type, u32 value)
 {
     Token token = {};
     token.m_type = type;
-    // FIXME: Avoid temporary StringBuilder here
```
Comment (linusg) [inline Userland/Libraries/LibWeb/CSS/Parser/Tokenizer.cpp:367]: pretty sure the StringBuilder is now dead code?

File: Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp [hunk 2]
```diff
@@ -562,7 +562,7 @@ Parser::ParseErrorOr<Selector::SimpleSelector> Parser::parse_pseudo_simple_selec
             }
             // FIXME: Support multiple, comma-separated, language ranges.
             Vector<DeprecatedFlyString> languages;
-            languages.append(pseudo_function.values().first().token().to_deprecated_string());
+            languages.append(pseudo_function.values().first().token().to_string().release_value_but_fixme_should_propagate_errors().to_deprecated_string());
```
Comment (linusg) [inline Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp:565]: > `pseudo_function.values().first().token().to_string().release_value_but_fixme_should_propagate_errors().to_deprecated_string()`

:yakkiss:

File: Userland/Libraries/LibWeb/CSS/Parser/Tokenizer.cpp (hunk 1 above)
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/CSS/Parser/Tokenizer.cpp:367]: :man_facepalming:

File: Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp (hunk 2 above)
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp:565]: It'll get better once other things are converted to String... (Or at least, that's what I tell myself. :sweat_smile: )

**Pull request merged**